includes/pages/admin_questions.php

```diff
@@ -153,13 +147,11 @@ function admin_questions()
                     return error('Incomplete call, missing Question ID.', true);
                 }
 
-                $question = DB::selectOne(
-                    'SELECT * FROM `Questions` WHERE `QID`=? LIMIT 1',
-                    [$question_id]
-                );
+                $questions =   Question::where(['id' => $question_id]);
+                $question = $questions->first();
                 if (!empty($question)) {
-                    DB::delete('DELETE FROM `Questions` WHERE `QID`=? LIMIT 1', [$question_id]);
-                    engelsystem_log('Question deleted: ' . $question['Question']);
+                    $questions->delete();
```

Comment:
This looks like you want to delete all of them, just using `$question->delete()` is really better

Reply:
I agree, as mentioned above this was due to basing the model on the old table.
I will refactor